dags/pull_here.py

```diff
@@ -55,12 +55,29 @@ def task_fail_slack_alert(context):
                         'LANG':'C.UTF-8'}
                 }
 
-dag = DAG('pull_here',default_args=default_args, schedule_interval='0 13 * * 1')
-# Run at 8 AM local time every monday
+dag = DAG('pull_here',default_args=default_args, schedule_interval=' 30 16 * * * ')
+#Every day at 1630
+# dag = DAG('pull_here',default_args=default_args, schedule_interval='0 13 * * 1')
+# Run at 1PM local time every monday
 
-t1 = BashOperator(
+# date_to_pull = datetime.today().date() - timedelta(days=2)
+
+# Execution date seems to be the day before this was run, so yesterday_ds_nodash
+# should be equivalent to two days ago. https://stackoverflow.com/a/37739468/4047679
+
+pull_data = BashOperator(
         task_id = 'pull_here',
-        bash_command = '/etc/airflow/data_scripts/.venv/bin/python3 /etc/airflow/data_scripts/here/traffic/here_api.py -d /etc/airflow/data_scripts/here/traffic/config.cfg', 
+        bash_command = '/etc/airflow/data_scripts/.venv/bin/python3 /etc/airflow/data_scripts/here/traffic/here_api.py -d /etc/airflow/data_scripts/here/traffic/config.cfg -s {{ yesterday_ds_nodash }} -e {{ yesterday_ds_nodash }} ', 
+        # params = {'date':date_to_pull.strftime('%Y%m%d')},
         retries = 0,
         dag=dag,
         )
+
+agg_tti = PostgresOperator(sql="SELECT covid.generate_citywide_tti('{{ yesterday_ds }}')",
+                           task_id='aggregate_tti',
+                           postgres_conn_id='here_bot',
+                           autocommit=True,
+                           retries = 0,
+                           dag=dag)
+
```

Comment:
Shouldn't this be in the analysis refresh dag?

Reply:
oh no, I missed that. Was only looking at miovision & rescu for covid related stuff.

Reply:
Where does `yesterday_ds` come from?